desktop/apps/editorial_features/components/venice_2017/client/index.coffee

```diff
@@ -37,19 +41,28 @@ module.exports = class VeniceView extends Backbone.View
       wrapAround: true
       initialIndex: sd.VIDEO_INDEX
     , (carousel) =>
-      flickity = carousel.cells.flickity
-      flickity.on 'settle', =>
-        @changeSection flickity.selectedIndex
+      @flickity = carousel.cells.flickity
+      @flickity.on 'settle', =>
+        @changeSection @flickity.selectedIndex
 
   changeSection: (i) ->
     @section = @curation.get('sections')[i]
+    @sectionIndex = i
     # Push route
     window.history.replaceState {}, i, '/venice-biennale/' + @section.slug
     # Swap video if it is published
     @swapVideo() if @section.published
     @swapDescription()
     @setupFollowButtons()
 
+  onNextVideo: ->
+    next = @sectionIndex + 1
+    if next is @curation.get('sections').length
+      next = 0
+    @flickity.select(next)
```

Comment:
Not a blocker, but I think this can be simplified to a one-liner by using Flickity's `next` method with a true flag to allow wrapping:
`@flickity.next(true)`